### ISSUE TITLE
How about the performance of attention masks which claim to localize the character

### ISSUE THREAD

**Issue by WenjiaWang0312** [chenjun2hao/Bert_OCR.pytorch#2]
Dear author, have you visualized the attention masks like the pics in the original paper? Can that really find the location of each character?
![image](https://user-images.githubusercontent.com/33180378/70685548-b4395980-1ce4-11ea-863e-c535496d0df5.png)
